src/properties.rs

```diff
@@ -67,9 +67,17 @@ where
         Ok(())
     }
 
-    /// Borrow configured interface for raw communication
-    pub fn borrow_iface_mut(&mut self) -> &mut DI {
-        &mut self.iface
+    /// Set the area on the display the data should be rendered onto
+    pub fn set_render_area(&mut self, start: (u8, u8), end: (u8, u8)) -> Result<(), ()> {
+        Command::ColumnAddress(start.0, start.1 - 1).send(&mut self.iface)?;
+        Command::PageAddress(start.1.into(), (end.1 - 1).into()).send(&mut self.iface)?;
+        Ok(())
+    }
+
+    /// Render data onto the display
+    pub fn render(&mut self, buffer: &[u8]) -> Result<(), ()> {
```

Comment:
Can we call this `draw()` or something? We're not really rendering anything, just sending a bunch of data to the display. It would also be worth mentioning in the doc comment that what is drawn depends on how `set_render_area()` is set.

Reply:
Certainly.